proto/zipkin_proto3/zipkin.proto

```diff
@@ -14,12 +14,16 @@
 
 syntax = "proto3";
 
+// This is the package for using protobuf with Zipkin API V2, but for historical
```

Comment:
Ok renamed the package but please let me have my social commentary here (don't mean to be snarky but I'm confident in a deep understanding of protobuf, much more than I may wish actually :P)

Reply:
freedom of expression allowed :)